ch_backup/clickhouse/control.py

```diff
@@ -300,8 +300,8 @@ def __init__(self, ch_ctl_config: dict, main_config: dict) -> None:
         self._shadow_data_path = os.path.join(self._root_data_path, "shadow")
         self._timeout = self._ch_ctl_config["timeout"]
         self._freeze_timeout = self._ch_ctl_config["freeze_timeout"]
-        self._system_unfreeze_timeout = self._ch_ctl_config["system_unfreeze_timeout"]
```

Comment:
Renamed `system_unfreeze_timeout` to `unfreeze_timeout` for consistency with `freeze_timeout`.